src/features/CareViewPatientsSummary/styles/CareViewPatientsSummary.scss

```diff
@@ -123,3 +123,15 @@
     }
   }
 }
+
+@media screen and (max-width: 1024px) {
```

Comment:
we could define mobile and tab width in SCSS global variable and use it over here so that if needed in future we just need to update the variables value